client/my-sites/checkout/checkout-thank-you/redesign-v2/test/is-refactored-for-thank-you-v2.js

```diff
@@ -88,4 +88,16 @@ describe( 'isRefactoredForThankYouV2', () => {
 			expect( isRefactoredForThankYouV2( props ) ).toBe( true );
 		}
 	} );
+
+	it( 'should return true for Jetpack Search', () => {
+		const props = {
+			receipt: {
+				data: {
+					purchases: [ { productSlug: 'jetpack_search' } ],
```

Comment:
Is targeting a specific slug for this test too fragile? Would it be preferable to loop through each of the [Jetpack Search product slugs](https://github.com/Automattic/wp-calypso/commit/77adc2c4bffa8059f47d9daad12a856a2b61a8e5)?

Reply:
Good question! This made me realize that I should have used `isSearch()` (function initially defined in `header.jsx` to check instead of `isJetpackSearchProductSlug()`. That function checks against `productType` instead of `productSlug`.

https://github.com/Automattic/wp-calypso/pull/88262/files#diff-5836f17692fa28c8c636e8d40f35daac248a6ee35e0f71130d05f9300605e1efL65